testdata/nml_with_too_long_edges.nml

```diff
@@ -2,7 +2,7 @@
 <things>
   <parameters>
     <experiment name="test_dataset" />
-    <scale x="11.0" y="11.0" z="25.0" />
+    <scale x="1.0" y="1.0" z="1.0" />
```

Comment:
Why did you change the scale in this (and in the other) nml file? The non-1-1-1-scale is more close to the real-world and therefore better for testing, I'd say.

Reply:
Well when I do that, I need to change all node positions in order since all of the nodes are testing certain cases and that would be changed. Shall I rather change all node positions?

Reply:
I see, yes, it would be cool if you could change the node positions instead.

Reply:
Done ^^